### PR TITLE
Rewrote regex section of tour

### PR DESCRIPTION
**Pull request by travissarles**

### CODE REVIEW

**Comment by travissarles** [comment]
@masseguillame

File: tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md [hunk 1]
```diff
@@ -12,36 +12,50 @@ next-page: extractor-objects
...
+    |\tmargin: 0;
+    |\theight: 108px;
+    |\twidth: 100%;""".stripMargin
+
+for (patternMatch <- keyValPattern.findAllMatchIn(input))
+  println(s"key: ${patternMatch.group(1)} value: ${patternMatch.group(2)}")
+```
+Here we parse out the keys and values of a String. Each match has a group of sub-matches. Here is the output:
+```
+key: tbackground-color value: #A03300
```
Comment (MasseGuillaume) [inline tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md]: \t

**Comment by MasseGuillaume** [comment]
I would link to https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html

There is more example in the api: https://www.scala-lang.org/api/current/scala/util/matching/Regex.html

File: tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md (hunk 1 above)
Comment (travissarles) [inline tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md]: For some reason it's printing out like I showed.

File: tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md [hunk 2]
```diff
@@ -12,36 +12,50 @@ next-page: extractor-objects
...
+key: tbackground-image value: url(img
```
Comment (MasseGuillaume) [inline tutorials/tour/_posts/2017-02-13-regular-expression-patterns.md]: t ^

**Pull request merged**